limacharlie/Manager.py

```diff
@@ -70,6 +70,8 @@ def __init__( self, oid = None, secret_api_key = None, environment = None, inv_i
             isRetryQuotaErrors (bool): if True, the Manager will attempt to retry queries when it gets an out-of-quota error (HTTP 429).
         '''
         # If an environment is specified, try to get its creds.
+        environment = "default"
+        print("this is environment ", environment)
```

Comment:
```suggestion
```

Debug code?

Reply:
my bad accidentally pushed a file I did not want to push, reverted change